Documentation.md

```diff
@@ -71,50 +71,53 @@ const coupons = await client.getProductCoupons(123);
 ```
 <a name="PaddleSDK+getProductPlans"></a>
 
-### client.getProductPlans(productID) ⇒ <code>Promise</code>
-Get the current list of plans for a subscription
+### client.getProductPlans([productID]) ⇒ <code>Promise</code>
+Get the current list of all plans or plans for a product
 
 **Kind**: instance method of [<code>PaddleSDK</code>](#PaddleSDK)  
 **Fulfil**: <code>object</code> - The plans list  
 
 | Param | Type |
 | --- | --- |
-| productID | <code>number</code> | 
+| [productID] | <code>number</code> | 
 
 **Example**  
 ```js
+const plans = await client.getProductPlans();
 const plans = await client.getProductPlans(123);
 ```
 <a name="PaddleSDK+getPlanUsers"></a>
 
-### client.getPlanUsers(planID) ⇒ <code>Promise</code>
-Get the current list of users for a subscription plan
+### client.getPlanUsers([planID]) ⇒ <code>Promise</code>
+Get the current list of all users or users for a subscription plan
 
 **Kind**: instance method of [<code>PaddleSDK</code>](#PaddleSDK)  
 **Fulfil**: <code>object</code> - The users list  
 
 | Param | Type |
 | --- | --- |
-| planID | <code>number</code> | 
+| [planID] | <code>number</code> | 
 
 **Example**  
 ```js
+const users = await client.getPlanUsers();
 const users = await client.getPlanUsers(123);
 ```
 <a name="PaddleSDK+getPlanPayments"></a>
 
-### client.getPlanPayments(planID) ⇒ <code>Promise</code>
-Get the list of payments for a subscription plan
+### client.getPlanPayments([planID]) ⇒ <code>Promise</code>
+Get the list of all payments or payments for a subscription plan
```

Comment:
This file should not be manually edited. It's generated by a lint staged task when the `sdk.js` file is changed.

This change here is not reflected in `sdk.js` for `getPlanPayments`. You'll need to edit that file only and regenerate both the `.d.ts` and the docs file with the tasks defined in `package.json`.

Reply:
Cool - done. Don't use jsdoc for my own projects so not done that before!